Hearthstone Deck Tracker/Utility/WotogCounterHelper.cs

```diff
@@ -24,6 +26,12 @@ public static class WotogCounterHelper
 		public static bool? YoggInDeck => DeckContains(CardIds.Collectible.Neutral.YoggSaronHopesEnd);
 		public static bool? ArcaneGiantInDeck => DeckContains(CardIds.Collectible.Neutral.ArcaneGiant);
 
+		public static bool PlayerSeenJade => Core.Game.PlayerEntity.HasTag(GameTag.JADE_GOLEM);
```

Comment:
`PlayerEntity` can be null at the beginning of the game. 
This needs to be `Core.Game.PlayerEntity?.HasTag(JADE_GOLEM) ?? false`

We also have a static using statement for GameTag in here, it can just be `JADE_GOLEM` instead of `GameTag.JADE_GOLEM`.